src/Randomizer.SMZ3.Tracking/AutoTracking/GameInteractor.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Randomizer.Shared;
+using Randomizer.SMZ3.Tracking.Configuration;
+using Randomizer.SMZ3.Tracking.VoiceCommands;
+
+namespace Randomizer.SMZ3.Tracking.AutoTracking
+{
+    public class GameInteractor
+    {
+        private readonly AutoTracker _autoTracker;
+        private readonly Tracker _tracker;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GameInteractor"/>
+        /// class.
+        /// </summary>
+        /// <param name="tracker">The tracker instance.</param>
+        /// <param name="autoTracker">The auto tracker to associate with this module</param>
+        public GameInteractor(Tracker tracker, AutoTracker autoTracker)
```

Comment:
I'd be more inclined to call this GameService, to be consistent with the services I'm writing in the other branch.